### PR TITLE
Add run-in-console support

### PR DESCRIPTION
**Pull request by loopyme**

### CODE REVIEW

File: console_run.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    parser.add_argument(
+        "-v", "--verbose", help="do not print log to console", action="store_true",
+    )
+
+    parser.add_argument(
+        "-l", "--log", help="save log to the log file", action="store_true",
+    )
+
+    # arg check
+    parser.add_argument(
```
Comment (0xSeanll) [inline console_run.py:44]: Why the validation of arguments is optional?

File: console_run.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        "-d",
+        "--disable-image",
+        help="do not show the video content",
+        action="store_true",
+    )
+
+    # log settings
+    parser.add_argument(
+        "-v", "--verbose", help="do not print log to console", action="store_true",
```
Comment (0xSeanll) [inline console_run.py]: I believe "verbose" means the opposite.

File: console_run.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+    # runtime behaviors
+    parser.add_argument(
+        "--headless", help="do not show the browser", action="store_true",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--disable-image",
+        help="do not show the video content",
```
Comment (0xSeanll) [inline console_run.py]: With this option, only  tags are disabled, videos are not, so it should be "do not show images".

**Review by 0xSeanll — CHANGES_REQUESTED** [review]
Your contribution is appreciated. However, there are some minor problems. Some of them are commented at corresponding line. Others things are as follow.

- Please change log messages (except exception and traceback) to Chinese, so that they conform with `Latiao.py`.
- Please update `README.md` as well.

Your pull request will be merged once these problems are dealt with.

File: console_run.py (hunk 1 above)
Comment (loopyme) [reply console_run.py:44]: Although check_args is not expensive now, future is full of possibilities. This parameter allows confident users to skip the check, and let errors thrown in subsequent operations.

**Comment by loopyme** [comment]
BTW, have you considered make it a package?

**Review by 0xSeanll — APPROVED** [review]

**Comment by 0xSeanll** [comment]
It doesn't make sense. This is not a library.

**Pull request merged**